### PR TITLE
update langchain to nougat to extract pdf in example

### PR DESCRIPTION
**Pull request by ZHIHANCHEN03**

### CODE REVIEW

**Comment by goldmermaid** [comment]
nit: there are still some `PyPDFLoader` in each notebook, can you remove them?

**Review by goldmermaid — APPROVED** [review]

File: example/transform/openai_pdf_source_10k_QA.ipynb [hunk 1]
```diff
@@ -135,8 +99,9 @@
     "from dotenv import load_dotenv\n",
     "import os\n",
     "import pandas as pd\n",
-    "from uniflow.flow.client import TransformClient\n",
+    "from uniflow.flow.client import TransformClient, ExtractClient\n",
     "from uniflow.flow.config import TransformOpenAIConfig\n",
+    "from uniflow.flow.config import ExtractPDFConfig, NougatModelConfig\n",
     "from uniflow.op.model.model_config import OpenAIModelConfig\n",
     "from langchain.document_loaders import PyPDFLoader\n",
```
Comment (SayaZhang) [inline example/transform/openai_pdf_source_10k_QA.ipynb:106]: shall we remove unused import?

File: example/transform/openai_pdf_source_10k_QA.ipynb [hunk 2]
```diff
@@ -229,32 +280,34 @@
...
-       " Context(context=\"Our DE&I focus extends beyond our workforce and includes our communities, which we support in a number of ways. We have \\ncommitted to investments that aim to address racial inequality and improve diversity and representation in our communities. W e \\nalso are leveraging our global scale to accelerate business diversity , including investing in business training programs for women \\nand increasing the proportion of services supplied by minority-owned businesses.\\nCOMPENSATION AND BENEFITS \\nNIKE's to\")]"
+       "[Context(context='In addition, failure to optimize inventory or staffing in our fulfillment network increases our net shipping cost by requiring long-zone or partial shipments. We and our co-sourcers may be unable to adequately staff our fulfillment network and customer service centers. For example, productivity across our fulfillment network currently is being affected by regional labor market and global supply chain constraints, which increase payroll costs and make it difficult to hire, train, and deploy a suf'),\n",
```
Comment (SayaZhang) [inline example/transform/openai_pdf_source_10k_QA.ipynb:298]: shall we use `pprint.pprint` to print output? And, I think printing 3-5 pieces of data is enough.

File: example/transform/openai_pdf_source_10k_summary.ipynb [hunk 3]
```diff
@@ -200,32 +290,34 @@
   },
   {
    "cell_type": "markdown",
+   "id": "8cf645d3",
```
Comment (SayaZhang) [inline example/transform/openai_pdf_source_10k_summary.ipynb:293]: shall we add `\` inside the `"""` statement to remove the initial space on each line?
[https://google.github.io/styleguide/pyguide.html](url)

File: example/transform/self_instruct_pdf_source.ipynb [hunk 4]
```diff
@@ -114,19 +125,104 @@
     "input_file = os.path.join(f\"{dir_cur}/data/raw_input/\", pdf_file)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "90151cb3-2071-4a32-8b1d-30d5ecd0d5f8",
+   "metadata": {},
+   "source": [
+    "Load and split the pdf"
```
Comment (SayaZhang) [inline example/transform/self_instruct_pdf_source.ipynb:133]: shall we add `###` before the content to highlight the comment?

File: example/transform/self_instruct_pdf_source.ipynb [hunk 5]
```diff
@@ -114,19 +125,104 @@
...
... someone they think we ought to meet, or send us an email proposing we grab coffee. At this point we have two options, neither of them good: we can meet with them, and lose half a day\\'s work; or we can try to avoid meeting them, and probably offend them. Till recently we weren\\'t clear in our own minds about the source of the problem. We just took it for granted that we had to either blow our schedules or offend people. But now that I\\'ve realized what\\'s going on, perhaps there\\'s a third option: to write something explaining the two types of schedule. Maybe eventually, if the conflict between the manager\\'s schedule and the maker\\'s schedule starts to be more widely understood, it will become less of a problem. Those of us on the maker\\'s schedule are willing to compromise. We know we have to have some number of meetings. All we ask from those on the manager\\'s schedule is that they understand the cost. **Thanks** to Sam Altman, Trevor Blackwell, Paul Buchheit, Jessica Livingston, and Robert Morris for reading drafts of this. **Related:** * How to Do What You Love\\n* Good and Bad Procrastination\\n* Turkish Translation\\n* French Translation\\n* Korean Translation']\n"
```
Comment (SayaZhang) [inline example/transform/self_instruct_pdf_source.ipynb:187]: same, shall we use pprint to print the output? And can we reduce the output length, it is too long here.

**Pull request merged**